### PR TITLE
Renaming Models chapter to "ORM and Database Access"

### PR DESCRIPTION
**Pull request by lorenzo**
Some people have trouble finding the docs for accessing the database
and it is totally unterstandable since it is not easy to associate
Models = ORM. Also we should move away from telling people that the Model
is reserved exclusively for database interaction.

### CODE REVIEW

**Comment by ravage84** [comment]
While I totally agree with you about associating Models with ORM and about the wrong meaning exclusivity for database interaction, I just want to point out that we should stick to the M-V-C terms, as they are very central and a orientation point for many - especially newbies.
So if I click now Models and then get a""ORM and Database Access" instead, I could get confused, too.

How about:

> Models, ORM and Database Access
> ############################
> 

And then first explain __what models mean in general without hinting towards data__  and then second explain how and with what data access is done in CakePHP.

May be we should link to or borrow from:
http://en.wikipedia.org/wiki/Conceptual_model_%28computer_science%29
http://en.wikipedia.org/wiki/Domain_model

**Comment by lorenzo** [comment]
No, I still think we should not have the word `Model` so prominently. It adds another term that should be learned or understood while looking for answers

File: en/orm.rst [hunk 1]
```diff
@@ -1,18 +1,17 @@
-Models
-######
+ORM and Database Access
```
Comment (markstory) [inline en/orm.rst]: Database Access & ORM?

Comment (lorenzo) [inline en/orm.rst]: @markstory that works for me as well!

**Comment by antograssiot** [comment]
I pushed the name modification + the fr version

**Comment by jhli** [comment]
I think "Models and Databases" is good, those knowing the term "ORM" already know that they should look into that chapter? It also fits into the navigation where it says Request/Response, then Controllers, then Views and well, then it should say Models. So either it should be "Models and Database" or "Models, Database & ORM".

**Pull request merged**